bin/is-website-vulnerable.js

```diff
@@ -6,91 +6,87 @@ const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
 const { prompt } = require('enquirer')
+const isUrl = require('is-url-superb')
+const chalk = require('chalk')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
 
-const main = async () => {
-  let url = process.argv[2]
-  debug(`received url argument: ${url}`)
+let url = process.argv[2]
+debug(`received url argument: ${url}`)
 
-  const isWindows = os.type() === 'Windows_NT'
-  const isJson = !!argv.json
-  const showProgressBar = !isJson && !isWindows
+const isWindows = os.type() === 'Windows_NT'
+const isJson = !!argv.json
+const showProgressBar = !isJson && !isWindows
 
-  if (!url) {
-    if (isJson) {
-      throw new Error('Please provide a URL of a website to scan')
-    } else {
-      console.error('Woops! You forgot to provide a URL of a website to scan.')
-
-      try {
-        const response = await prompt({
-          type: 'input',
-          name: 'url',
-          message: 'Please provide a URL to scan:',
-          validate: input => input && input.length > 0
-        })
-        url = response.url
-      } catch (_) {
-        throw new Error('URL input prompt failed')
+;(async () => {
```

Comment:
(I added `;` here because of JS semicolon insertion, as otherwise the engine thinks were trying to invoke `isWindows` as a function, with the main async function being its parameter.)

Reply:
👍